package.json

```diff
@@ -383,7 +383,8 @@
     "swc-loader": "^0.2.6",
     "prettier": "2.8.8",
     "puppeteer": "19.6.0",
-    "ws": "8.17.1"
+    "ws": "8.17.1",
+    "**/@mdx-js/react/@types/react": "^17"
```

Comment:
this is specified as `>=16` which resolves to latest - in our case v18, that is not what we want though.